chat-plugins/poll.js

```diff
@@ -312,6 +312,7 @@ exports.commands = {
 	pollhelp: ["/poll allows rooms to run their own polls. These polls are limited to one poll at a time per room.",
 				"Accepts the following commands:",
 				"/poll create [question], [option1], [option2], [...] - Creates a poll. Requires: % @ * # & ~",
+				"/poll htmlcreate [question], [option1], [option2], [...] - Creates a poll with HTML. Requires: % @ * # & ~",				
```

Comment:
This line has trailing whitespace at the end of it, which is why Travis isn't passing